wagtail/core/migrations/0068_rename_data_json_modellogentry_data_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2022-02-22 04:41
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0067_alter_modellogentry_data_json_and_more'),
```

Comment:
We can squash the migrations if desired.

Reply:
Yep, I think combining the migrations would be good.